_posts/2020-04-30-insights.adoc

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: 'Quarkus Insights on Youtube Live'
+date: 2020-04-30 
+tags: annoucement insights
+synopsis: Quarkus Insights is a live streaming channel where we and guests will talk to get insights on how Quarkus works and how it is used.
+author: maxandersen
+---
+
+Yesterday, Emmanual Bernard and I kicked of https://www.youtube.com/watch?v=YKm8rVzLhNE[Quarkus Insights #0: Inception]. All based on https://twitter.com/maxandersen/status/1254451547842531329[this tweet] where ~76% was up for us doing "something" in video/podcast'y style.
+
+It is going to be recorded live as part of our existing https://youtube.com/c/quarkusio[QuarkusIO] channel. To be notified when new shows are planned and when they go live https://youtube.com/c/quarkusio[Go Subscribe] and hit the bell for notifications!
+
+In this first episode we introduces the idea and concepts and took questions from the ~35 guests that made it in with short notice!
+
+video::YKm8rVzLhNE[youtube,width=640, height=360]
+
+It was super-fun and looking forward to do it again on Monday.
+
+https://youtu.be/OCPFdpvL1Q0[Quarkus Insights episode, #1: Tests] with a guest appearance from Georgios Andrianakis.
```

Comment:
We need to explain what will be discussed and that live demos will be there. Otherwise I don't know why I would show up